routes/localIp.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const { getLocalIps } = require("./controllers/localIp.controller");
+const router = express.Router();
```

Comment:
```suggestion
const router = express.Router();
const { getLocalIps } = require("./controllers/localIp.controller");
```
사용한 순서에 맞게 위와 같이 되면 좀 더 보기 좋을 것 같습니다.